app/proguard-rules.pro

```diff
@@ -19,3 +19,13 @@
 # If you keep the line number information, uncomment this to
 # hide the original source file name.
 #-renamesourcefileattribute SourceFile
+
+-keepattributes *Annotation*, InnerClasses
+-dontnote kotlinx.serialization.SerializationKt
+-keep,includedescriptorclasses class io.github.wax911.retgraph.**$$serializer { *; } # <-- change package name to your app's
+-keepclassmembers class com.yourcompany.yourpackage.** { # <-- change package name to your app's
```

Comment:
@Andre-Sequeira Do you know what to use here? This is a forked library with the original package name, but it says "change package name to your app's", so do you think it should be com.tonsser, com.tonsser.tonnser, com.tonsser.domain, or maybe even remove it completely?

Reply:
Don't know. But it either is the lib pack name or it is not needed at all. Having the Tonsser package name doesn't make sense in this lib.